.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Kubernetes-sigs/krew CI
+on: 
+  push:
+    branches: 
+      - '*'
+    tags: 
+      - 'v*.*.*'
+  pull_request:
+    branches: 
+      - '*'
+  release:
+    types: [published, created, edited]
```

Comment:
I think we don't need this if we do `push.tags` ?
Can you explain how it helps?

Reply:
When editing the releases either from the GitHub interface, the pipeline will also run, but I also see that it is not necessary. I already removed it.